docs/tasks/istio-installation.md

```diff
@@ -1,37 +1,105 @@
 ---
-title: Installing Istio
-headline: Installing Istio
+title: Istio Installation
+headline: Istio Installation
 sidenav: doc-side-tasks-nav.html
 bodyclass: docs
 layout: docs
 type: markdown
 ---
 {% capture overview %}
-This task shows how to install Istio in a Kubernetes cluster. You'll learn
+This page shows how to get started using Istio in a Kubernetes cluster. You'll learn
```

Comment:
"This task shows you how to install Istio in a .. "
Because we are re-using this task for everything (not just getting started). i think this is probably a remnant from an older version of the docs. :)